lib/logstash/devutils/rspec/spec_helper.rb

```diff
@@ -28,13 +25,15 @@
 end
 
 $TESTING = true
-if RUBY_VERSION < "1.9.2"
-  $stderr.puts "Ruby 1.9.2 or later is required. (You are running: " + RUBY_VERSION + ")"
-  raise LoadError
+if RUBY_VERSION < "2.3"
+  raise LoadError.new("Ruby >= 2.3.0 or later is required. (You are running: " + RUBY_VERSION + ")")
 end
 
-if ENV["TEST_DEBUG"]
-  LogStash::Logging::Logger::configure_logging("WARN")
+if level = (ENV["TEST_DEBUG"] || ENV['LOGGER_LEVEL'])
+  logger, level = level.split('=') # 'logstash.filters.grok=DEBUG'
```

Comment:
> if we are going to allow specifying a specific logger's level, should we provide a way to specify multiple?

might be too much - better going for log4j.properties or simply `"DEBUG"` for all.
you usually test a single plugin -> the env varaible is meant as an assistance.

> should we configure logging with OFF before doing setting overrides, to ensure we start with a clean slate? what is the intended behaviour of other loggers when a single named logger is set to debug?

for my use-cases it was enough to either set all or a single logger e.g. when debugging grok `logstash.filters.grok=DEBUG` (LS' logging defaults where good enough of a starting point), 
I do not like the default `OFF` that much since at some point I got lost for a while and eventually managed to find an error somewhere in the pipeline that was otherwise silent due the default OFF. here's what my typical usage looks like:

`LOGGER_LEVEL=logstash.outputs.file=INFO LOGSTASH_SOURCE=1 jruby -rbundler/setup -S rspec spec/outputs/file_spec.rb`

... but if you feel like this is somehow incomplete I am willing to put more effort it as needed.

Reply:
- you're right, supporting multiple isn't necessary now, and we can always add it in later if we need to.
- +1 to the "silent" default being `ERROR`